src/components/radio/__tests__/RadioGroup.test.tsx

```diff
@@ -11,20 +11,26 @@ describe('components/radio/RadioGroup', () => {
         sandbox.verifyAndRestore();
     });
 
-    const renderRadioButtons = (onChange?: Function) =>
-        mount(
+    const renderRadioButtons = (onChange?: Function, children?: Array<React.ReactElement> | React.ReactElement) => {
+        children = children || [
+            <RadioButton
+                key="1"
+                data-resin-target="resin1"
+                description="radio1desc"
+                label="Radio Button 1"
+                value="radio1"
+            />,
+            <RadioButton key="2" label="Radio Button 2" value="radio2" />,
+            <RadioButton key="3" description="radio3desc" label="Radio Button 3" value="radio3" />,
+            <RadioButton key="4" label="Radio Button 4" value="radio4" />,
+        ];
+
+        return mount(
```

Comment:
Can we use `shallow` here?

Reply:
I think it might've been deliberate to use `mount` originally since the implementation depends heavily on the radio input element. I'm not sure if there's a good way to migrate some of these existing tests to `shallow`